nook-desktop.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Name=Nook
+Exec=camp.nook.nookdesktop
```

Comment:
there is no such executable. it should be what you put in `command` in the manifest.